src/library/cmd_reader.h

```diff
@@ -35,31 +35,10 @@ namespace xlang::cmd
 
             auto last{ options.end() };
 
-            for (C i = 1; i < argc; ++i)
-            {
-                std::string_view arg{ argv[i] };
-
-                if (arg[0] == '-')
-                {
-                    arg.remove_prefix(1);
-                    last = find(options, arg);
-
-                    if (last == options.end())
-                    {
-                        throw_invalid("Option '-", arg, "' is not supported");
-                    }
-
-                    m_options.try_emplace(last->name);
-                }
-                else if (last == options.end())
-                {
-                    throw_invalid("Value '", arg, "' is not supported");
-                }
-                else
-                {
-                    m_options[last->name].push_back(std::string{ arg });
-                }
-            }
+			for (C i = 1; i < argc; ++i)
```

Comment:
The code still has white space issues all over the place. Find the "show white space" option in your editor and turn it on. Then change your editor settings to use a tab size of 4 and to insert tabs as spaces. Then cut and paste the entire source file and it should reset the white space consistently. Confirm visually.